libretro_core_options.h

```diff
@@ -175,16 +175,16 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(filter_exclude_sprite),
       "Exclude Sprites from Filtering",
       NULL,
-      "Do not apply texture filtering to sprites. Prevent seams in various games with 2D sprite-rendered backgrounds. Use together with Adaptive Smoothing or another post-processing filter for best effect.",
+      "Do not apply texture filtering to sprites. Prevents seams in various games with 2D sprite-rendered backgrounds. Use together with Adaptive Smoothing or another post-processing filter for best effect.",
```

Comment:
Possibly the "various" word is unneeded. It should be obvious that the affected games are those with 2D sprite-rendered backgrounds, whether it's all of those games or not. Or maybe it could be changed to "certain" or "specific".

Reply:
Yeah, you're right, that word is unnecessary fluff.